File: ELearningFrontEnd/src/app/dtos/question-answer/question-answer.req.ts
export class QuestionAnswerReq{

    content:string;
    lectureId:number;
    userId:number;
    questionId:number | null;

    constructor(data: any) {
        this.content = data.content;
        this.lectureId = data.lectureId;
        this.userId = data.userId;
        this.questionId = data.questionId;
    }
}

File: ELearningBackEnd/src/main/java/com/example/elearningbackend/question_answer/QuestionAnswerReq.java
package com.example.elearningbackend.question_answer;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class QuestionAnswerReq {

    private String content;

    private long lectureId;

    private long userId;

    private Long questionId;

    private boolean isActivated;
}
